tests/test_2024_07_02.py

```diff
@@ -0,0 +1,230 @@
+"""
+Tests for voting 24/01/2023.
+"""
+
+import math
+
+from brownie.network.account import LocalAccount
+
+from scripts.vote_2024_07_02 import start_vote
+from utils.config import contracts
+from utils.mainnet_fork import chain_snapshot
+from utils.test.deposits_helpers import fill_deposit_buffer
+from utils.test.event_validators.staking_router import validate_staking_module_update_event, StakingModuleItem
+from utils.test.simple_dvt_helpers import simple_dvt_add_keys
+from utils.test.tx_tracing_helpers import *
+from utils.test.event_validators.payout import (
+    validate_token_payout_event,
+    Payout,
+)
+from brownie.network.transaction import TransactionReceipt
+
+TOTAL_BASIS_POINTS = 10_000
+NODE_OPERATORS_REGISTRY_ID = 1
+SIMPLE_DVT_ID = 2
+
+agent_addr = "0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c"
+expected_sdvt_module = StakingModuleItem(
+    id=SIMPLE_DVT_ID,
+    name="Simple DVT Module",
+    address=None,
+    target_share=400,
+    module_fee=800,
+    treasury_fee=200,
+)
+
+expected_payout = Payout(
+    token_addr="0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32",
+    from_addr=agent_addr,
+    to_addr="0x17F6b2C738a63a8D3A113a228cfd0b373244633D",
+    # https://docs.lido.fi/multisigs/lido-contributors-group#41-pool-maintenance-labs-ltd-pml
+    amount=96_666.62 * (10**18),  # 96666.62 LDO in wei,
```

Comment:
I would also use the `int` here instead of `float`. At least to avoid the editor types error:
<img width="556" alt="image" src="https://github.com/lidofinance/scripts/assets/14151334/1e1114b1-cbc1-4ddb-9faf-80f1b8e6c877">